spec/fixtures/files/vacancy_sources/ark.xml

```diff
@@ -34,6 +34,7 @@
         https://arkcareers.engageats.co.uk/Vacancies/W/1972/0/247720/19522/tutor-fellow-primary-or-secondary
       </engAts:rssFeedVacancyUrl>
       <pubDate>Tue, 09 Mar 2021 00:00:00 GMT</pubDate>
+      <visaSponsorshipAvailable>true</visaSponsorshipAvailable>
```

Comment:
I have strong suspicions that, if coming, the field will come as part of the `engAts` fields. I think I recall when the value is a custom value added for the Schools to fill outside the standard provided fields, it comes as `<engAts:field>`, but we cannot know 100% until they send it (if they decide to).

Reply:
Ah I see. I am happy to go with your suspicions!